packages/frontend/editor-ui/src/components/canvas/elements/CanvasThinkingPill.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
```

Comment:
Since this is purely UI component it would make sense to add it to design library